pkg/server/cloud/aws/instances.go

```diff
@@ -231,6 +232,25 @@ func bootImageSpecToDescribeImagesInput(spec cloud.BootImageSpec) *ec2.DescribeI
 	return input
 }
 
+func getRootDeviceName(img *ec2.Image) (string, error) {
+	if img.RootDeviceName == nil {
+		msg := "cannot get root device name from image: " + *img.Name
+		return "", errors.New(msg)
+	}
+	return *img.RootDeviceName, nil
```

Comment:
The preferred way is to use `aws.String()` and `aws.StringValue()` for dealing with pointers in the data structures in the AWS SDK.

Reply:
Thanks :+1:  switched to using those.